src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/EventSourceTestParser.cs

```diff
@@ -658,7 +658,7 @@
 //                case 0:
 //                    return x;
 //                default:
-//                    Debug.Assert(false, "Bad field index");
+//                    Debug.Fail("Bad field index");
```

Comment:
this comment says "This code was automatically generated by the TraceParserGen tool" .. unless that tool is updated, whereever it is, the comment probably shouldn't change.

Reply:
Seems like the whole file could just be deleted. @brianrob?

Reply:
Looks like it's all comments, so yes.  @davmason?

Reply:
https://github.com/dotnet/runtime/pull/105237

Reply:
👍Just found it as I catch up on e-mail.